<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basic_operation</class>
 <widget class="QMainWindow" name="basic_operation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>692</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>947</width>
    <height>692</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>12</pointsize>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="btn_layout">
      <item>
       <widget class="QPushButton" name="dac_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>DAC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sampling_rate_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Sampling Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="input_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filter_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="corners_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Corners</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DAC Mode</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dataset</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Passthrough</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>DAC Repetitions</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Infinity</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;=</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QTextEdit" name="textEdit">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;=</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>2,147,483,647</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPing_CyDAQ"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPing_CyDAQ">
   <property name="text">
    <string>Ping CyDAQ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
